CHANGES/6722.feature

```diff
@@ -0,0 +1 @@
+Added ``InvalidRedirectURL`` which is now raised instead ``ValueError`` or ``InvalidURL`` when redirect url is invalid
```

Comment:
```suggestion
Added an ``InvalidRedirectURL`` exception which is now raised instead of
:py:exc:`ValueError` or ``InvalidURL`` when the redirect URL
is invalid -- by :user:`setla`.
```